.dockerignore

```diff
@@ -60,3 +60,6 @@ packages/*/crates/*/pkg
 
 # vagrant metadata
 /scripts/vm-nat-testing/.vagrant
+
+# make sure anything which is vendored is copied
+!/vendor/**/*.log
```

Comment:
This PR starts vendoring all cargo dependencies. That means they become part of our Git repo in a fixed version. They need to be available in Docker builds because otherwise our builds would not be able to run. Once built, they can be removed though from the final Docker image, which is what is done in the upcoming PR by @robertkiel

Reply:
More specifically for the log, cargo vendoring creates checksums for the whole package, therefor all files must be present.